charts/cluster-api-runtime-extensions-nutanix/templates/ccm/aws/manifests/aws-ccm-v1.27.10-configmap.yaml

```diff
@@ -7,7 +7,7 @@
 #=================================================================
 apiVersion: v1
 data:
-  aws-ccm-v1.30.3.yaml: |
+  aws-ccm-v1.27.10.yaml: |
```

Comment:
This is a bit confusing. The git diff should show upgrading from 1.27.9 to 1.27.10 ?

Reply:
Oh very strange, it "renaming" incorrect files.
Let me see if I can fix that.

Reply:
Split the commits, but that didn't fix the overall PR view.